### ISSUE TITLE
 Minor display bug on progress bars

### ISSUE THREAD

**Issue by jrynd** [erosson/swarm#406]
It appears as if the text in the progress bar has a "shadow" behind it.

Well, when the progress bar moves at a certain speed, the "shadow" and the white text don't progress at the same time. The text moves right slightly before the shadow does, creating an annoying "flickering" effect. This only happens at certain progress speeds and with certain refresh intervals.

**Comment by jrynd** [comment]
Specifically: The shadow moves half a pixel, then the text moves a whole pixel, then the shadow moves half a pixel.

This is on Chrome on Windows.

**Comment by erosson** [comment]
What theme are you using? I don't see this when using Slate or the default. Chrome/Windows. Any browser zoom?

**Comment by jrynd** [comment]
Slate, zoom is at 90%. haven't tried it at 100%, will let you know.

**Comment by jrynd** [comment]
Doesn't happen when zoom is at 100%.

**Comment by erosson** [comment]
Still don't see this, sorry. Tried a few different zoom levels and a few different progressbar speeds. What progressbar speed/time-remaining are you seeing it with?

> with certain refresh intervals.

Do you mean monitor refresh rate, or the game's fps setting?

**Comment by jrynd** [comment]
Game was set at 10 fps.Progressbar took around 20 minutes. Happens over
whole length of progressbar.

**Comment by jrynd** [comment]
No, actually it doesn't. It only happens when the progress bar is near the middle.
